### PR TITLE
use a constant for the activejob message attributes

### PR DESCRIPTION
**Pull request by timbreitkreutz**
Co-authored-by: Tamara Temple <tamara.temple@drip.com>
Co-authored-by: Norma Farah <norma.farah@drip.com>

This is an experimental branch to see if we can eliminate the HB from https://dripcom.atlassian.net/browse/CI-1171

### CODE REVIEW

**Comment by timbreitkreutz** [comment]
Sorry for the noise

File: lib/shoryuken/extensions/active_job_adapter.rb [hunk 1]
```diff
@@ -88,6 +79,13 @@ def perform(_sqs_msg, hash)
           Base.execute hash
         end
       end
+
+      MESSAGE_ATTRIBUTES = {
+        'shoryuken_class' => {
+          string_value: JobWrapper.to_s,
+          data_type: 'String'
+        }
+      }.freeze
```
Comment (phstc) [inline lib/shoryuken/extensions/active_job_adapter.rb:88]: That's a good catch. constant makes sense.

Comment (timbreitkreutz) [reply lib/shoryuken/extensions/active_job_adapter.rb:88]: Thanks @phstc  -- We'll put together a better PR a bit later.  Turns out a frozen constant messes up our code downstream.

**Comment by timbreitkreutz** [comment]
Nicer version has been submitted: #635

**Pull request closed without merging**